tests/testCountInputs.py

```diff
@@ -0,0 +1,207 @@
+        exp.setXY0(afwGeom.Point2I(offset, offset))
+        exp.setWcs(wcsRef)
+        exp.setPsf(afwDetection.GaussianPsf(5, 5, 1.0))
+        exp.getInfo().setCoaddInputs(afwImage.CoaddInputs(afwTable.ExposureTable.makeMinimalSchema(),
+                                                          afwTable.ExposureTable.makeMinimalSchema()))
+        # Set the fake CCDs that "went into" making this coadd, using the differing wcs objects created
+        # above.
+        ccds = exp.getInfo().getCoaddInputs().ccds
+        for wcs in wcsList:
+            record = ccds.addNew()
+            record.setWcs(wcs)
+            record.setBBox(imageBox)
+            record.setValidPolygon(Polygon(afwGeom.Box2D(imageBox)))
+
+        # Add the stars and other associated data into the exposure object
+        exp.getMaskedImage().getImage().set(0)
+        exp.getMaskedImage().getMask().set(0)
+        exp.getMaskedImage().getVariance().set(1.0)
+        for pp in positions:
+            x, y = map(int, pp)
+            exp.getMaskedImage().getImage().set0(x, y, value)
+            exp.getMaskedImage().getMask().set(x - offset, y - offset, value)
+
+        # Configure a singleFrameMeasurementTask to run CountInputs
+        measureSourcesConfig = measBase.SingleFrameMeasurementConfig()
+        measureSourcesConfig.plugins.names = ["base_NaiveCentroid", "base_CountInputs"]
+        measureSourcesConfig.slots.centroid = "base_NaiveCentroid"
+        measureSourcesConfig.slots.psfFlux = None
+        measureSourcesConfig.slots.apFlux = None
+        measureSourcesConfig.slots.modelFlux = None
+        measureSourcesConfig.slots.instFlux = None
+        measureSourcesConfig.slots.calibFlux = None
+        measureSourcesConfig.slots.shape = None
+        measureSourcesConfig.validate()
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        task = measBase.SingleFrameMeasurementTask(schema, config=measureSourcesConfig)
+        catalog = afwTable.SourceCatalog(schema)
+        # Set information to create the "fake" stars
+        for pp in positions:
+            foot = afwDetection.Footprint(afwGeom.Point2I(pp), 1.0)
+            peak = foot.getPeaks().addNew()
+            peak.setIx(int(pp[0]))
+            peak.setIy(int(pp[1]))
+            peak.setFx(pp[0])
+            peak.setFy(pp[1])
+            peak.setPeakValue(value)
+
+            source = catalog.addNew()
+            source.setFootprint(foot)
+        
+        task.run(catalog, exp)
+        # Loop over the postions of each star and calculate how many CCD boxes each is in, and compare
+        # this number to what is returned from the SingleFrameMeasurementTask
+        for i, pp in enumerate(positions):
+            number = sum(afwGeom.Box2D(imageBox).contains(wcs.skyToPixel(wcsRef.pixelToSky(pp))) for
+                         wcs in wcsList)
+            self.assertEqual(catalog[i].get("base_CountInputs_value"), number)
+
+        if showPlot:
```

Comment:
The convention in the other test cases is to use `if display` together with a try/except block:
```
try:
    display
except NameError:
    display = False
```
Probably a good idea to adopt the same idiom here.